### PR TITLE
Add Github related extensions in devcontainer

### PR DESCRIPTION
**Pull request by yp05327**

### CODE REVIEW

**Comment by yp05327** [comment]
It seems that GitHub Actions Extension not works well for the forked repo. It won't show any running jobs from the origin repo.
Maybe I need to do something to link them?

**Comment by yp05327** [comment]
image
This extension can display suggestions in vscode!

**Review by silverwind — APPROVED** [review]

**Comment by yp05327** [comment]
image
Merged branch have a red point, it is easy to manage your branches. 😄

**Review by lunny — APPROVED** [review]

File: .gitpod.yml [hunk 1]
```diff
@@ -35,6 +35,7 @@ vscode:
     - ms-azuretools.vscode-docker
     - zixuanchen.vitest-explorer
     - qwtel.sqlite-viewer
+    - GitHub.vscode-pull-request-github
```
Comment (yardenshoham) [inline .gitpod.yml:38]: I think Gitpod has this anyway

**Pull request merged**